src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -359,6 +359,135 @@
+        },
+        {
+            "path": "/v1/debug/bundle/create",
+            "operations": [
+                {
+                    "method": "POST",
+                    "summary": "Start creating a debug bundle",
+                    "type": "void",
+                    "nickname": "start_debug_bundle",
+                    "produces": [
+                        "application/json"
+                    ],
+                    "parameters": [
+                        {
+                            "name": "logs-since",
+                            "in": "query",
+                            "required": false,
+                            "type": "string"
+                        },
+                        {
+                            "name": "logs-until",
+                            "in": "query",
+                            "required": false,
+                            "type": "string"
+                        },
+                        {
+                            "name": "logs-size-limit",
+                            "in": "query",
+                            "required": false,
+                            "type": "string"
+                        },
+                        {
+                            "name": "metrics-interval",
+                            "in": "query",
+                            "required": false,
+                            "type": "string"
+                        }
+                    ],
+                    "responses": {
+                        "200": {
+                            "description": "Successfully started debug bundle"
+                        },
+                        "500": {
+                            "description": "Internal Server error"
+                        }
+                    }
+                }
+            ]
+        },
+        {
+            "path": "/v1/debug/bundle/status",
+            "operations": [
+                {
+                    "method": "HEAD",
+                    "summary": "Returns the status of the debug bundle",
+                    "type": "string",
+                    "nickname": "get_debug_bundle_status",
+                    "produces": [
+                        "application/json"
+                    ],
+                    "parameters": [],
+                    "responses": {
+                        "200": {
+                            "description": "The debug bundle is completed and ready for download"
+                        },
+                        "404": {
+                            "description": "The debug bundle is running but not yet ready"
+                        },
+                        "410": {
+                            "description": "The debug bundle is not running or it is not on disk"
+                        },
+                        "500": {
+                            "description": "Internal Server error"
+                        }
+                    }
+                }
+            ]
```

Comment:
I would stick closer to the PRD here.  I think what we want to do is add a `HEAD` operation to `/v1/debug/bundle` because they return pretty much the same information.  The difference is that `GET` will return the debug bundle if it's available.

Reply:
This is not a `GET`. It is a `HEAD`.

Reply:
I'm saying don't create a separate endpoint just for `HEAD`.  Add `HEAD` to the endpoint that returns the file.